app/jenkins-for-jira-ui/src/common/analytics/analytics-client.ts

```diff
@@ -61,20 +54,18 @@ export class AnalyticsClient {
 				}
 			);
 
-			this.getContext(userType, tenantType);
+			await this.getContext(userType, tenantType);
 		}
 
-		const baseAttributes: BaseAttributes = {
```

Comment:
Killed this off because, while we use this in GH4J, it's not needed here because of the context we're passing.